### PR TITLE
fix: update logo

### LINKED ISSUE #3912: Update logo in these locations
Browser tab favicon for both Dev and Cloud:

https://www.figma.com/file/rrcVngskc3p6mIIYg1jH8Y/Rill-Visual-Identity?type=design&node-id=612-146232&mode=design&t=qt9F9MbAQwKZCQzj-11

Rill dev welcome page

https://www.figma.com/file/Qt6EyotCBS3V6O31jVhMQ7/RILL?type=design&node-id=1987-35610&mode=design&t=BkgjZVfS7TyqoZIL-11

### PR DESCRIPTION
**Pull request by mindspank**
Updates:

- Favicons for rill developer, cloud and docs
- Welcome background splash, logo and tiles.

Not fixing the "Welcome to Rill" text gradient as I think @bcolloran and @jkhwu are already discussing that one?

Closes #3912

### CODE REVIEW

File: web-common/static/img/welcome-bg-art.png
Comment (ericpgreen2) [inline web-common/static/img/welcome-bg-art.png:1]: This image looks a little pixelated on my large monitor. I've just asked @jkhwu if she can make a higher-res version.

**Comment by bcolloran** [comment]
looks like this will fix #3899 as well.

we could also close #3903 if we address `<RillLogo>`-- although, when I search for `<RillLogo`, I only find instances of `<RillLogoSquareNegative`, so perhaps `<RillLogo>` is never used, in which case I think we should just delete that component to avoid confusion later on

**Review by ericpgreen2 — APPROVED** [review]

**Comment by mindspank** [comment]
Closes #3903

File: web-common/static/img/welcome-bg-art.png
Comment (mindspank) [reply web-common/static/img/welcome-bg-art.png:1]: Updated assets to x2.

**Pull request merged**